null/data/batch_metrics_enabled.conf

```diff
@@ -0,0 +1 @@
+true
```

Comment:
Can we remove these ```null/*/*.conf``` files? They are generated during test execution. Maybe adding ```null/*``` to ```.gitignore``` will help avoid these issues in the future.

Reply:
ack